pyproject.toml

```diff
@@ -1,2 +1,161 @@
+[project]
+name = "tinytuya"
+dynamic = ["version", "readme"]
+authors = [
+    { name = "Jason Cox", email = "jason@jasonacox.com" },
+]
+description = "Python module to interface with Tuya WiFi smart devices"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+dependencies = [
+    "colorama",
+    "requests",
+]
+
+requires-python = ">= 3.8"
```

Comment:
I think this is fine. This aligns to end of life with python version (https://devguide.python.org/versions/)

@uzlonewolf  thoughts?